macros/src/main/scala/com/bcf/statefun4s/generic/FlinkFunction.scala

```diff
@@ -0,0 +1,133 @@
+package com.bcf.statefun4s.generic
```

Comment:
Did you go to school to comment your code like this? Oh yeah, YOU DIDNT GO TO SCHOOL!

Reply:
LOL!

Reply:
Is that what they taught you in economics class?